test/suites/incremental_copy.sh

```diff
@@ -30,7 +30,7 @@ do_copy() {
   incus storage volume set "${source_pool}" container/c1 user.foo=main
 
   # Set size to check this is supported during copy.
-  incus config device set c1 root size=50MiB
+  incus config device set c1 root size=200MiB
```

Comment:
We should get the non-linstor changes out of here so they can be pushed to LTS and reduce the delta (even if only really needed under Linstor).